algopy_testing/src/algopy/primitives/biguint.py

```diff
@@ -0,0 +1,135 @@
+from __future__ import annotations
+
+import functools
+from typing import TYPE_CHECKING
+
+from algopy.constants import MAX_UINT512
+from algopy.primitives.bytes import Bytes
+
+if TYPE_CHECKING:
+    from algopy.primitives.uint64 import UInt64
+
+# TypeError, ValueError are used for operations that are compile time errors
+# ArithmeticError and subclasses are used for operations that would fail during AVM execution
+
+
+@functools.total_ordering
+class BigUInt:
+    """
+    A python implementation of an AVM 64-bit unsigned integer
```

Comment:
Should either remove this (as end users will normally only see the stub documentation) or update the description to reflect biguint (512-bit unsigned integer)

Reply:
It has been updated.